tests/api/v1/datadog/api_usage_metering_test.go

```diff
@@ -259,6 +259,40 @@ func TestUsageSNMP(t *testing.T) {
 	assert.True(usage.HasUsage())
 }
 
+func TestUsageBillableSummary(t *testing.T) {
+	ctx, finish := WithRecorder(WithTestAuth(context.Background()), t)
+	defer finish()
+	assert := tests.Assert(ctx, t)
+
+	startDate, endDate := getStartEndMonth(ctx)
+	usage, httpresp, err := Client(ctx).UsageMeteringApi.GetUsageBillableSummary(ctx).StartDate(startDate).EndDate(endDate).Execute()
+	if err != nil {
+		t.Errorf("Error getting Usage Billable Summary: Response %s: %v", err.(datadog.GenericOpenAPIError).Body(), err)
+	}
+	assert.Equal(200, httpresp.StatusCode)
+	assert.True(usage.HasUsage())
+
+	var usageItem = usage.GetUsage()[0]
+	assert.Equal("Logs Probe - Test", usageItem.GetOrgName())
+	assert.Equal("Pro", usageItem.GetBillingPlan())
+	assert.Equal("927176c4b", usageItem.GetPublicId())
+	assert.Equal(time.Date(2020, 06, 01, 0, 0, 0, 0, time.UTC), usageItem.GetStartDate().UTC())
```

Comment:
What is the relation between `time.Date(2020, 06, 01, 0, 0, 0, 0, time.UTC)` and `startDate`?